src/screens/FinancialSummary/FinancialSummaries.tsx

```diff
@@ -0,0 +1,151 @@
+import React from 'react';
```

Comment:
Sorry 1 more thing. To keep the naming convention consistent with `Customers` (which has `CustomerMain`), can we rename this `FinancialSummariesMain`? I think I goofed on the file name when I told you to rename to `FinancialSummaries` instead of `FinancialSummariesMain`, sorry about the extra work!